app/views/RoomActionsView/index.js

```diff
@@ -655,8 +656,8 @@ class RoomActionsView extends React.Component {
 
 	renderJitsi = () => {
 		const { room } = this.state;
-		const { jitsiEnabled } = this.props;
-		if (!jitsiEnabled || room.teamMain) {
+		const { jitsiEnabled, jitsiEnableTeams } = this.props;
+		if (!jitsiEnabled || (room.teamMain && !jitsiEnableTeams)) {
```

Comment:
Missing validation in case jitsi is or is not enabled for channels

Reply:
I'd think on refactoring this whole logic.
Instead of thinking what's necessary to `return null`, I'd write the opposite.
Writing affirmative propositions are always easier to read than negative.
Something like this might work.
`(room.teamId && jitsiEnableTeams && jitsiEnabled) || (!room.teamId && jitsiEnabled)`